src/ebay-segmented-buttons/button.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react'
+import classNames from 'classnames'
+import { excludeComponent, findComponent } from '../common/component-utils'
+import { EbayIcon } from '../ebay-icon'
+import { SegmentedButtonProps } from './types'
+
+const SegmentedButton: FC<SegmentedButtonProps> = ({
+    selected,
+    children,
+    onClick,
```

Comment:
We need to support `className` also and merge with `segmented-buttons__button`, without it it will overwrite the original one